edb/server/dbview/dbview.pxd

```diff
@@ -124,6 +124,7 @@ cdef class DatabaseConnectionView:
         Database _db
         bint _query_cache_enabled
         object _protocol_version
+        public bint is_internal  # internal dbview doesn't have a frontend connection
```

Comment:
Maybe `is_transient`? Since IIRC it can also apply to a query from the HTTP protocol, which is not really internal, but *also* isn't long lived.